Jenkinsfile.github

```diff
@@ -121,6 +100,28 @@ pipeline {
                         }
                     }
                 }
+
+                stage("API Documentation") {
+                    environment {
+                        GIT_AUTHOR_NAME = "Jenkins"
+                        GIT_AUTHOR_EMAIL = "jenkins@algol60.net"
+                        GIT_COMMITTER_NAME = "Jenkins"
+                        GIT_COMMITTER_EMAIL = "jenkins@algol60.net"
+                        EMAIL = "jenkins@algol60.net"
+                    }
+                    steps {
+                        withCredentials(
+                            [usernamePassword(credentialsId: credentialsId, usernameVariable: 'ARTIFACTORY_USERNAME', passwordVariable: 'ARTIFACTORY_TOKEN')]
+                        ) {
+                            script {
+                                env.GITHUB_APP_INST_TOKEN = getGitHubToken(appId: "330087", installId: "37198173", credentialsId: "github-jenkins-auto-push-bot-key")
+                                sh """
+                                    ./hack/gen-push-swagger-markdown.sh ${env.DOCS_CSM_BRANCH} ${env.TAG_NAME ? '--push --wait' : '--push --wait'}
```

Comment:
@phalseth-hpe The `/hack/gen-push-swagger-markdown.sh` script can also be run periodically (i.e. daily) with `--push` option to ensure that head of `docs-csm` is always tagged. I'll leave thos for another PR - if we need this for 1.3, we'll need to add REST API generation to 1.3, or simplify it to skip REST API and only produce a tag.